uint8 start_red
uint8 start_green
uint8 start_blue
uint8 end_red
uint8 end_green
uint8 end_blue
float32 transform_time 
float32 run_time
